test/NuGet.Core.Tests/NuGet.Common.Test/PathResolverTests.cs

```diff
@@ -669,12 +683,15 @@ private void PopulateTestDirectory()
             DirectoryInfo directory3 = Directory.CreateDirectory(System.IO.Path.Combine(directory2.FullName, "dir3"));
             DirectoryInfo directory4 = Directory.CreateDirectory(System.IO.Path.Combine(directory1.FullName, "dir4"));
             Directory.CreateDirectory(System.IO.Path.Combine(rootDirectory.FullName, "dir5"));
+            var directory6 = Directory.CreateDirectory(System.IO.Path.Combine(rootDirectory.FullName, "dir6"));
```

Comment:
Shouldn't this be DirectoryInfo as well?

Reply:
Yes, since previous lines uses `DirectoryInfo`